packages/opentelemetry-web/src/utils.ts

```diff
@@ -230,3 +230,75 @@ export function parseUrl(url: string): HTMLAnchorElement {
   a.href = url;
   return a;
 }
+
+/**
+ * Get element XPath
+ * @param target
+ * @param optimised
+ */
+export function getElementXPath(target: any, optimised?: boolean) {
```

Comment:
What does the `optimized?` parameter do? Could you add an explanation for it in the JSDoc above?